### ISSUE TITLE
p2 Update site: Missing org.eclipse.fx.core

### ISSUE THREAD

**Issue by kerner1000** [BestSolution-at/FX-Test#23]
Hi, I wanted to add the p2 site to my target platform, but get the following error:

![tareget](https://user-images.githubusercontent.com/13817521/29860689-1d975ca2-8d67-11e7-958f-6f0a053cbac4.png)

**Comment by tomsontom** [comment]
you also have to add https://download.eclipse.org/efxclipse/runtime-nightly/site/ this is where org.eclipse.fx.core is coming from

**Comment by kerner1000** [comment]
Cannot get it to work. Retried a several times..

![nightly](https://user-images.githubusercontent.com/13817521/29865184-2be9d7f4-8d75-11e7-8327-813142c0b84f.png)

**Comment by tomsontom** [comment]
sorry should be http://download.eclipse.org/efxclipse/runtime-nightly/site/ not "https"

**Comment by tomsontom** [comment]
I'm not sure btw what you are trying to test? Is your OSGi-App written without e(fx)clipse runtime-libs? From the e(fx)clipse you best choose the `org.eclipse.fx.target.feature`

**Comment by kerner1000** [comment]
Ok, http seems to work. Actually I think efxclipse should be already available.
See screenshot, its our local target platform.

Its not nightly though.

And maybe its a problem that those features are local and not fetched from p2.

![fxclipse](https://user-images.githubusercontent.com/13817521/29875285-cf25f2b2-8d99-11e7-8cf6-94e9fdd95823.png)

Motivation: We have an efxclipse application and I would like to test some UI related stuff. Currently I was only looking for the FXTestRule/ FXTestRunner to execute Unit tests on the FXApplication thread.

**Comment by tomsontom** [comment]
The target you show looks like an IDE-Target and not a JavaFX-runtime one, it also is **2.4.0**. Test-FX requires **3.0.0**

**Comment by kerner1000** [comment]
Besides the version missmatch, does the IDE not include the runtime?

**Comment by tomsontom** [comment]
No, not necessarily. The IDE only comes with the runtime-modules it uses itself this might or might not include fx.core (indeed fx.core is there) but others like fx.ui.controls are eg not there.

**Comment by kerner1000** [comment]
OK thanks a lot! I guess it is sufficient to replace ide\*2.4.0 with ide\*3.0.0 files and add core\*3.0.0 as well?

**Comment by tomsontom** [comment]
I think what we should do is to add extra feature/category with the e(fx)clipse upstream dependencies
